ci-operator/step-registry/rosa/cluster/wait-ready/cluster/rosa-cluster-wait-ready-cluster-commands.sh

```
@@ -198,6 +199,25 @@ while true; do
           FAILED_INSTALL="yes"
           break
         fi
+
+        # Provisioner launch detection: for Classic clusters, check if Hive ever started
+        if [[ "${HOSTED_CP}" != "true" ]] && [[ "${CLUSTER_STATE}" == "installing" ]]; then
+          infra_id_check=$(cat ${cluster_info_json} | jq -r '.infra_id')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)